fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -318,17 +318,22 @@ private RdfStream getResourceTriples(final int limit, final FedoraResource resou
 
         final List<Stream<Triple>> streams = new ArrayList<>();
 
+
         if (returnPreference.getValue().equals("minimal")) {
             streams.add(getTriples(resource, of(PROPERTIES, MINIMAL)).filter(tripleFilter));
 
-            if (ldpPreferences.prefersServerManaged()) {
+            // Mementos already have the server managed properties in the PROPERTIES category
+            // since mementos are immutable and these triples are no longer managed
```

Comment:
This probably has more to do with the Memento creation than return. But doesn't this mean you can't get a Memento without server managed triples?
I did a test and a normal (no options) request for the OriginalResource vs a Memento match up perfectly. 

But if you add a `Prefer: return=representation; omit="http://fedora.info/definitions/v4/repository#ServerManaged"` then the Memento has 3 triples while the OriginalResource has 1.

<img width="943" alt="original_vs_memento" src="https://user-images.githubusercontent.com/2857697/46085645-118ecc80-c16c-11e8-998a-a81f6aa96f45.png">

Reply:
Yes, I think you're right and it alls about what context you're in. I think this turns into a meta conversation quickly.  The memento has the server managed triples from the original, now immutable, but does it also have it's own server managed triples?  Those original triples are no longer server managed (though they used to be).   It seems the decision was made that those original server managed triples are no longer server managed (makes sense, since it's a copy in time). 
Mostly I had to "say ^^ out loud" to process it myself.  
I think you've found a good twist on this...   why does the original only return one, when it seems like it should have those other two as well? Weird.

Reply:
after thinking about this some... it seems we have the issue of what server managed  triple is part of the original resource (which seems like being in PROPERTY is okay) and what server managed triples belong to the memento.  Ie, the memento should have it's own server managed triples, but we can't distinguish, in the returned object, which server managed triples belong to the memento itself and which belong to the copy of the resource the memento is for.  ... so, should the memento have it's own server managed triples?

Reply:
And how would we represent that?  Should the returned object have a subject for the original resource and then another set of triples where the subject is the  memento, which contain the server managed triples for the memento?

Reply:
@dbernstein @awoods    what do you guys think of the above questions?

Reply:
@whikloj - I'm actually confused as to why we're not getting those other triples in the original, since I think the only server managed properties are:  
        fedora:createdBy
        fedora:created
        fedora:lastModified  
        fedora:lastModifiedBy

Reply:
I'm sorry for opening this can of metaphysical worms. I'm not sure what is considered valid here.

The Memento with `omit` server managed triples gives this:
```
<http://localhost:8080/rest/versioned>
        rdf:type  ldp:RDFSource ;
        rdf:type  ldp:Container ;
        rdf:type  ldp:BasicContainer .
```

The same Memento with `include` server managed triples gives this:
```
<http://localhost:8080/rest/versioned>
        rdf:type               fedora:Container ;
        rdf:type               ldp:RDFSource ;
        rdf:type               ldp:Container ;
        rdf:type               fedora:Resource ;
        rdf:type               ldp:BasicContainer ;
        fedora:created         "2018-09-17T15:17:22.936Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> ;
        fedora:writable        true ;
        fedora:lastModified    "2018-09-17T15:17:22.936Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> ;
        fedora:lastModifiedBy  "fedoraAdmin" ;
        fedora:createdBy       "fedoraAdmin" .
```

For the OriginalResource, `omit`ting server managed triples:
```
<http://localhost:8080/rest/versioned>
        rdf:type  ldp:BasicContainer .
```

and `include`ing server managed triples:
```
<http://localhost:8080/rest/versioned>
        rdf:type               fedora:Container ;
        rdf:type               fedora:Resource ;
        rdf:type               ldp:BasicContainer ;
        fedora:lastModifiedBy  "fedoraAdmin" ;
        fedora:createdBy       "fedoraAdmin" ;
        fedora:created         "2018-09-17T15:17:22.936Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> ;
        fedora:lastModified    "2018-09-17T15:17:22.936Z"^^<http://www.w3.org/2001/XMLSchema#dateTime> ;
        rdf:type               ldp:RDFSource ;
        rdf:type               ldp:Container ;
        fedora:writable        true .
```

So maybe the problem is that the OriginalResource is considering these as server managed triples?
```
<>    rdf:type               ldp:RDFSource ;
      rdf:type               ldp:Container .
```

Reply:
@whikloj :  thanks for the clarification of the current behavior.   I've put the topic on the agenda for tomorrow's call.